### DISCUSSION TITLE
My friend built a RISC-V processor in Terraria

### DISCUSSION THREAD

**Discussion by kxing28** [NUS-CG3207/labs#30] [Other Stuff]
I took this course because of this video made by a classmate. After completing the course, I can't believe he did what we did without using any HDL. Enjoy the video!
[![YouTube](http://i.ytimg.com/vi/zXPiqk0-zDY/hqdefault.jpg)](https://www.youtube.com/watch?v=zXPiqk0-zDY)

**Comment by r-panicker** [comment]
Simply wow! This is mind-blowing stuff.
Thanks for linking it here.
